src/webparts/reactAllGroups/loc/en-us.js

```diff
@@ -1,6 +1,6 @@
 define([], function() {
```

Comment:
Can you review that string in the mystring file match string in the en and fr file.

Reply:
I think I already made changes in the upgrading branch to match. I will make sure in this branch too.